libethcore/ChainOperationParams.cpp

```diff
@@ -35,6 +36,14 @@ ChainOperationParams::ChainOperationParams():
 }
 
 EVMSchedule const& ChainOperationParams::scheduleForBlockNumber(u256 const& _blockNumber) const
+{
+    if (_blockNumber >= lastForkBlock)
+        return lastForkWithAdditionalEIPsSchedule;
+    else
+        return fixedScheduleForBlockNumber(_blockNumber);
```

Comment:
(Nit) Would a better name for this function be something like "forkScheduleforBlockNumber"? As someone who's unfamiliar with this part of the code I don't know what a "fixed" schedule is 😀

Reply:
yes, will change